documentation/votes.http

```diff
@@ -0,0 +1,10 @@
+###
+POST http://localhost:8080/voting/proposal
```

Comment:
We need to agree with this because I used curl for the demos. Or maybe we can use openapi

Reply:
Let's use markdown and HTTP requests both